### PR TITLE
feat: created ctrl and routes block/unblock, seedr for account, config suquelize

### PR DESCRIPTION
**Pull request by Vladosik12333**

### CODE REVIEW

File: src/controllers/account.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const createUserModel = require("../models/account");
+const { DataTypes } = require("sequelize");
+
+const { db } = require("../databaseSetup");
+
+const Account = createUserModel(db, DataTypes);
+
+const blockAccount = async (req, res) => {
+  const { userId } = req.params;
+  console.log(userId);
```
Comment (VadymKharyton) [inline src/controllers/account.js:10]: Remove console.log

File: src/index.js [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
-const unitofmeasurings = require('./routes/unitofmeasurings.js');
+const eventsRouter = require("./routes/events");
+const accountRouter = require("./routes/account");
+const unitofmeasurings = require("./routes/unitofmeasurings.js");
 const adminContactsRoutes = require("./routes/contactAdmin");
 
 const app = express();
 
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short';
+const formatsLogger = app.get("env") === "development" ? "dev" : "short";
```
Comment (VadymKharyton) [inline src/index.js:17]: Replace all double quotes with single ones

File: src/controllers/account.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  await Account.update(
+    { isBlocked: 1 },
+    {
+      where: {
+        userId: userId,
```
Comment (VadymKharyton) [inline src/controllers/account.js:16]: Simplify to single userId

Fix same issue in other places

**Review by VadymKharyton — COMMENTED** [review]
Changes look good, just added some comments to clean up this code

File: src/index.js (hunk 2 above)
Comment (Vladosik12333) [reply src/index.js:17]: oh, okay

File: src/controllers/account.js (hunk 3 above)
Comment (Vladosik12333) [reply src/controllers/account.js:16]: I agree

File: src/controllers/account.js (hunk 1 above)
Comment (Vladosik12333) [reply src/controllers/account.js:10]: yes

File: src/index.js (hunk 2 above)
Comment (Vladosik12333) [reply src/index.js:17]: why it need? We always use double.

Comment (Vladosik12333) [reply src/index.js]: fixed

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**